projects/bp-graphql/src/operations/deleteExhibitionSection.ts

```diff
@@ -0,0 +1,16 @@
+import { Operation, graphql } from '../Operation.js';
+import { checkExhibition } from '../isAllowedHelpers.js';
+
+export default {
+  group: 'exhibition',
+  isAllowed: checkExhibition('id'),
+  document: graphql`
+    mutation deleteExhibitionSection($id: ID!) {
```

Comment:
When this PR is merged, we will have to remember to perform some manual database migration to grant these new permissions (this one and `deleteExhibitionPicture`) to the relevant users.